### PR TITLE
updated to latest support matrix and description

### PR DESCRIPTION
**Pull request by gnardiello65**
updated to latest support matrix and description

### CODE REVIEW

File: app/_hub/cleafy/cleafy/index.md [hunk 1]
```diff
@@ -1,96 +1,81 @@
...
+      required: "no"
+      default: '',
+      value_in_examples: ""
+      description: "Specifies the type of encoding accepted from the backend server. This plugin does not support gzip-encoded requests."
 
-### Cleafy-Stream Kong plugin
+# Cleafy-Stream Kong plugin
 
-### Installation
+# Installation
```
Comment (h3xar0n) [inline app/_hub/cleafy/cleafy/index.md]:
```suggestion
### Installation
```

File: app/_hub/cleafy/cleafy/index.md [hunk 2]
```diff
@@ -1,96 +1,81 @@
...
 
 Plugin installation mainly depends on where your Kong installation runs:
 
-#### Standalone Installation
-Once the .rock file has been obtained from your Cleafy distributor it can be installed using the luarocks package manager.
+## Standalone Installation
```
Comment (h3xar0n) [inline app/_hub/cleafy/cleafy/index.md]:
```suggestion
#### Standalone Installation
```

File: app/_hub/cleafy/cleafy/index.md [hunk 3]
```diff
@@ -1,96 +1,81 @@
...
+Once the .rock file has been obtained from youir Cleafy diustributor it can be installed using the luarocks package manager.
 ```
 luarocks install cleafy-plugin-for-kong-VERSION.rock
 ```
 
-### Typical configuration (via curl)
+## Typical configuration (via curl)
```
Comment (h3xar0n) [inline app/_hub/cleafy/cleafy/index.md]:
```suggestion
### Typical configuration (via curl)
```

File: app/_hub/cleafy/cleafy/index.md [hunk 4]
```diff
@@ -1,96 +1,81 @@
...
-  # https://opensource.org/licenses/alphabetical
+license_type: "Included in the standard Cleafy license"
 
-privacy_policy_url: https://www.iubenda.com/privacy-policy/31282315
+privacy_policy_url: "https://www.iubenda.com/privacy-policy/31282315"
 
 kong_version_compatibility:
   community_edition:
     compatible:
-      - 0.13.x
```
[the remark this answers is not in the document]
Comment (Sessa93) [reply app/_hub/cleafy/cleafy/index.md]: Yes, I just updated it to the latest supported version. Do we need to specify that the previous version are still supported?

File: app/_hub/cleafy/cleafy/index.md [hunk 5]
```diff
@@ -1,96 +1,81 @@
...
 
 params:
-  name: cleafy
-  api_id: true
-  service_id: true
-  consumer_id: true
-  route_id: true
+  name: "cleafy-plugin-for-kong"
+  api_id: false
+  service_id: false
```
[the remark this answers is not in the document]
Comment (Sessa93) [reply app/_hub/cleafy/cleafy/index.md]: The plugin is applied to a route not to a service. I have updated _route_id_

File: app/_hub/cleafy/cleafy/index.md [hunk 6]
```diff
@@ -1,96 +1,81 @@
 ---
 name: Cleafy plugin for Kong
-publisher: Cleafy
```
[the remark this answers is not in the document]
Comment (Sessa93) [reply app/_hub/cleafy/cleafy/index.md]: FIxed

[the remark this answers is not in the document]
Comment (GayleNeumann) [reply app/_hub/cleafy/cleafy/index.md]: @Sessa93 Thank you for making updates as h3xar0n requested. We have not posted your updates yet, as there are some breaking changes when posting your updates to the Kong Plugins Hub. I am looking into this issue and will get back to you when we have a resolution, which I believe is template related.

File: app/_hub/cleafy/cleafy/index.md [hunk 7]
```diff
@@ -50,25 +49,12 @@ params:
...
-# Beneath the next --- use Markdown (redcarpet flavor) and HTML formatting only.
-#
-# The remainder of this file is for free-form description, instruction, and
-# reference matter.
-# Your headers must be Level 3 or 4 (parsing to h3 or h4 tags in HTML).
-# This is represented by ### or #### notation preceding the header text.
-###############################################################################
-# BEGIN MARKDOWN CONTENT
----
+      description: Specifies the type of encoding accepted from the backend server. This plugin does not support gzip-encoded requests.
```
Comment (lena-larionova) [inline app/_hub/cleafy/cleafy/index.md]:
```suggestion
---
```
Missing the closing --- lines for the metadata. That should fix the output.

File: app/_hub/cleafy/cleafy/index.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (lena-larionova) [reply app/_hub/cleafy/cleafy/index.md]: Yes, you need to specify all supported versions. See the sidebar for this plugin, for example: https://docs.konghq.com/hub/kong-inc/ldap-auth/

File: app/_hub/cleafy/cleafy/index.md [hunk 8]
```diff
@@ -50,26 +60,13 @@ params:
       default:
       value_in_examples: http://cleafy.com
       description: Specifies the URL pointing to the Cleafy API.
+
```
Comment (lena-larionova) [inline app/_hub/cleafy/cleafy/index.md]: Remove this blank line

File: app/_hub/cleafy/cleafy/index.md [hunk 9]
```diff
@@ -32,16 +31,27 @@ kong_version_compatibility:
...
 params:
-  name: cleafy
-  api_id: true
-  service_id: true
-  consumer_id: true
+  name: cleafy-plugin-for-kong
+  api_id: false
+  service_id: false
+  consumer_id: false
   route_id: true
```
Comment (lena-larionova) [inline app/_hub/cleafy/cleafy/index.md]: Remove this blank line

File: app/_hub/cleafy/cleafy/index.md [hunk 10]
```diff
@@ -50,26 +60,13 @@ params:
...
-# Beneath the next --- use Markdown (redcarpet flavor) and HTML formatting only.
-#
-# The remainder of this file is for free-form description, instruction, and
-# reference matter.
-# Your headers must be Level 3 or 4 (parsing to h3 or h4 tags in HTML).
-# This is represented by ### or #### notation preceding the header text.
-###############################################################################
-# BEGIN MARKDOWN CONTENT
+      description: Specifies the type of encoding accepted from the backend server. This plugin does not support gzip-encoded requests.
 ---
```
Comment (lena-larionova) [inline app/_hub/cleafy/cleafy/index.md]: Add blank lines on either side of this ---

**Comment by lena-larionova** [comment]
@Sessa93 yaml formatting is really finicky and the extra/missing blank lines are still messing with it. I left comments for the specific lines, but here's what it should look like when you're done:

Screen Shot 2020-03-19 at 10 01 00 AM

**Pull request merged**